resources/asciidoctor/lib/extensions.rb

```diff
@@ -6,6 +6,9 @@
 require_relative 'elastic_compat_preprocessor/extension'
 require_relative 'elastic_include_tagged/extension'
 
+# This extensions is special because it is evil - just loading it is enough
+require_relative 'inline_callout/extension'
```

Comment:
Then is it possible to only even load it when `inline-callouts` is defined?

Reply:
It'd be possible but strange because we load these before we have a document to check what is defined. We'd have to load late which is indeed strange.